test/integration.js

```diff
@@ -46,7 +46,7 @@ describe('integration tests:', function () {
             cb(undefined, item);
           });
         vorpal
-          .command('overwrite me')
+          .command('overwrite_me')
```

Comment:
But I notice there are many test failing because the same assumption, that 'overwrite me' (e.g.) could be a single command name. So I fear I'm misunderstanding some vorpal assumption here. 
@dthree could you help me?